hook/hook_test.go

```diff
@@ -0,0 +1,67 @@
+// Copyright (C) 2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package hook_test
+
+import (
+	"testing"
+
+	"github.com/ava-labs/avalanchego/vms/components/gas"
+	"github.com/ava-labs/libevm/core/types"
+	"github.com/ava-labs/libevm/params"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/ava-labs/strevm/gastime"
+	. "github.com/ava-labs/strevm/hook"
```

Comment:
(nit) I'll reconfigure the linter later, but please put this in its own group, at the end. It fundamentally changes how the file should be interpreted, so I think it should be made more obvious to a reader.